pkg/web/apps/apps_psql_repository_test.go

```diff
@@ -164,3 +172,64 @@ func Test_appsPostgreSQLRepository_GetAppVersions_WillReturnNoAppVersions(t *tes
 		t.Fatalf("Expected 0 apps to be returned from the database, got %v", len(*versions))
 	}
 }
+
+
+//
+//func Test_appsPostgreSQLRepository_UpdateAllVersions(t *testing.T) {
```

Comment:
@craicoverflow I am stuck trying to mock the database here. Could you give me a hand? What is wrong?

Reply:
Hi @camilamacedo86.

I had a look and it appears you are calling `mock.ExpectQuery` where you should be using `mock.ExpectExec` because you are doing an UPDATE to the database.

```go
mock.ExpectExec(getUpdateAllAppVersionsQueryString).WithArgs(msg, status, id).WillReturnResult(sqlmock.NewResult(1, 1))
```

Note ^: I am not sure if the arguments in `sqlmock.NewResult(1, 1)` are correct.

You also need to make a call to the actual repository:

```go
a := NewPostgreSQLRepository(db)
err = a.UpdateAllAppVersions(id, msg, status)
```